subsys/bluetooth/Kconfig.logging

```diff
@@ -0,0 +1,1083 @@
+	select DEPRECATED
+	help
+	  Use this option to enable Low Power debug logs for the
+	  Bluetooth mesh functionality.
+
+config BT_MESH_DEBUG_FRIEND
+	bool "[DEPRECATED] Friend debug"
+	select DEPRECATED
+	help
+	  Use this option to enable Friend debug logs for the
+	  Bluetooth mesh functionality.
+
+config BT_MESH_DEBUG_PROXY
+	bool "[DEPRECATED] Proxy debug"
+	select DEPRECATED
+	depends on BT_MESH_GATT
+	help
+	  Use this option to enable Proxy protocol debug logs.
+
+config BT_MESH_DEBUG_SETTINGS
+	bool "[DEPRECATED] Persistent settings debug"
+	select DEPRECATED
+	depends on BT_SETTINGS
+	help
+	  Use this option to enable persistent settings debug logs.
+
+config BT_MESH_DEBUG_CDB
+	bool "[DEPRECATED] Configuration database debug"
+	select DEPRECATED
+	depends on BT_MESH_CDB
+	help
+	  Use this option to enable configuration database debug logs.
+
+config BT_MESH_DEBUG_CFG
+	bool "[DEPRECATED] Configuration debug"
+	select DEPRECATED
+	help
+	  Use this option to enable node configuration debug logs for the
+	  Bluetooth mesh functionality.
+
+endmenu # [DEPRECATED] Mesh
+
+menu "[DEPRECATED] Services"
+
+# IAS
+
+config BT_DEBUG_IAS_CLIENT
+	bool "[DEPRECATED] Immediate Alert Service Client debug"
+	select DEPRECATED
+	depends on BT_IAS_CLIENT
+	help
+	  This option enables enables Immediate Alert Service Client debug logs.
+
+# OTS
+
+# TODO: Unify logging for OTS server and client - the client uses
+# "BT" debugging, the server does not.
+
+config BT_DEBUG_OTS_CLIENT
+	bool "[DEPRECATED] Object Transfer Service Client debug"
+	select DEPRECATED
+	help
+	  Use this option to enable Object Transfer Client debug logs
+
+endmenu # [DEPRECATED] Services
+
+endif # BT_DEPRECATED_LOG
+
+# (subsys/bluetooth/Kconfig)
+
+module = BT
+module-str = "Bluetooth"
+source "subsys/logging/Kconfig.template.log_config"
+
+# COMMON (subsys/bluetooth/common/Kconfig)
+
+module = BT_HCI_DRIVER
+legacy-debug-sym = BT_DEBUG_HCI_DRIVER
+module-str = "Bluetooth HCI driver"
+source "subsys/bluetooth/common/Kconfig.template.log_config_bt"
+
+module = BT_RPA
+legacy-debug-sym = BT_DEBUG_RPA
+module-str = "Bluetooth Resolvable Private Address (RPA)"
+source "subsys/bluetooth/common/Kconfig.template.log_config_bt"
+
+menu "Audio"
+
+# AICS
+
+module = BT_AICS
+legacy-debug-sym = BT_DEBUG_AICS
+module-str = "Audio Input Control Service"
+source "subsys/bluetooth/common/Kconfig.template.log_config_bt"
+
+module = BT_AICS_CLIENT
+legacy-debug-sym = BT_DEBUG_AICS_CLIENT
+module-str = "Audio Input Control Service client"
+source "subsys/bluetooth/common/Kconfig.template.log_config_bt"
+
+# BAP
+
+module = BT_BAP_STREAM
+legacy-debug-sym = BT_BAP_DEBUG_STREAM
+module-str = "Bluetooth Audio Stream"
+source "subsys/bluetooth/common/Kconfig.template.log_config_bt"
+
+parent-module = BT
+module = BT_AUDIO_CODEC
+module-str = "Bluetooth Audio Codec"
+source "subsys/logging/Kconfig.template.log_config_inherit"
+
+module = BT_ASCS
+legacy-debug-sym = BT_DEBUG_ASCS
```

Comment:
IMO we should just remove all the legacy handling in these as well, and then just do the proper switch now (for LE Audio)